MAUI/Cartesian-Charts/Transform-axis-value-to-pixel-value-and-vice-versa.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Transform axis value to pixel value and vice-versa | Syncfusion
+description: Learn here all about to transform axis value to pixel value and vice-versa in SfCartesianChart in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Transform axis value to pixel value and vice-versa
+
+[`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1) offers two utility methods to transform the pixel to chart point and vice-versa.
+
+* [`ValueToPoint(ChartAxis axis,double value)`]() - Converts the data point value to screen point.
+* [`PointToValue(ChartAxis axis,double x, double y)`]() - Converts the screen point to chart value.
+
+{% highlight c# %}
+
+float chartPoint = chart.ValueToPoint(axis, value);
```

Comment:
`protected override void OnTouchDown(ChartBase chart, float pointX, float pointY)
{
    base.OnTouchDown(chart, pointX, pointY);

    if (chart is SfCartesianChart cartesianChart)
    {
        //Convert screen point to chart point.
        var xValue = cartesianChart.PointToValue(cartesianChart.XAxes[0], pointX, pointY);
        var yValue = cartesianChart.PointToValue(cartesianChart.YAxes[0], pointX, pointY);

        //Convert chart point to screen point.
        var xPoint = cartesianChart.ValueToPoint(cartesianChart.XAxes[0], xValue);
        var yPoint = cartesianChart.ValueToPoint(cartesianChart.YAxes[0], yValue);
        
    }
}
`

Reply:
Change code snippet.